app/kubernetes-ingress-controller/1.3.x/references/annotations.md

```diff
@@ -0,0 +1,528 @@
+with the Ingress resource. This controls the [matching evaluation
+order](/../../latest/proxy/#evaluation-order) for regex-based
+routes. It accepts any integer value. Routes are evaluated in order of highest
+priority to lowest.
+
+Sample usage:
+
+```yaml
+konghq.com/regex-priority: "10"
+```
+
+Please note the quotes (`"`) around the integer value.
+
+### konghq.com/methods
+
+> Available since controller 0.9
+
+Sets the `methods` setting on the Kong route associated with the Ingress
+resource. This controls which request methods will match the route. Any
+uppercase alpha ASCII string is accepted, though most users will use only
+[standard methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods).
+
+Sample usage:
+
+```yaml
+konghq.com/methods: "GET,POST"
+```
+
+### konghq.com/snis
+
+> Available since controller 1.1
+
+Sets the `snis` match criteria on the Kong route associated with this Ingress.
+When using route-attached plugins that execute during the certificate 
+phase (for example, [Mutual TLS Authentication](/hub/kong-inc/mtls-auth/)), 
+the `snis` annotation allows route matching based on the server name 
+indication information sent in a client's TLS handshake.
+
+Sample usage:
+
+```yaml
+konghq.com/snis: "foo.example.com, bar.example.com"
+```
+
+### konghq.com/request-buffering
+
+> Available since controller 1.2
+
+Enables or disables request buffering on the Kong route associated with this
+Ingress.
+
+Sample usage:
+
+```yaml
+konghq.com/request-buffering: "false"
+```
+
+### konghq.com/response-buffering
+
+> Available since controller 1.2
+
+Enables or disables response buffering on the Kong route associated with this
+Ingress.
+
+Sample usage:
+
+```yaml
+konghq.com/response-buffering: "false"
+```
+
+### konghq.com/host-aliases
+
+> Available since controller 1.3
+
+Set addtional hosts for routes created from rules on this Ingress
+
+Sample usage:
+
+```yaml
+konghq.com/host-aliases: "example.com,example.net"
+```
+
+Note that this annotation applies to all rules equally. An Ingress like this:
+
+```
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: example-ingress
+  annotations:
+    konghq.com/host-aliases: "example.com,example.net"
+spec:
+  rules:
+  - host: "foo.example"
+    http:
+      paths:
+      - pathType: Prefix
+        path: "/bar"
+        backend:
+          service:
+            name: service1
+            port:
+              number: 80
+  - host: "bar.example"
+    http:
+      paths:
+      - pathType: Prefix
+        path: "/bar"
+        backend:
+          service:
+            name: service2
+            port:
+              number: 80
+```
+
+Will result in two routes:
+
+```
+{"hosts":["foo.example", "example.com", "example.net"], "paths":["/foo"]}
+{"hosts":["bar.example", "example.com", "example.net"], "paths":["/bar"]}
+
+Take care not to accidentally create unintentionally overlapping routes by not
+including the same path in multiple rules.
```

Comment:
```suggestion

{:.important}
> To avoid creating overlapping routes, don't reuse the same path in multiple rules.
```
I hope I understood what you were going for correctly, the number of negatives was a bit confusing. Got rid of both "accidentally" and "unintentional", as they're adding complexity - intentionality is up to the user to decide, and we should just warn them about the possibility and how to avoid it.